app/server/api/mock_data_api.py

```diff
@@ -0,0 +1,72 @@
+from flask import Blueprint, request, make_response, jsonify, g
+from flask.views import MethodView
+
+from server import db
+from server.utils.auth import requires_auth
+from server.models.transfer_usage import TransferUsage
+from server.models.user import User
+from server.utils.mock_data import create_payments, create_disbursments, create_users
+
+
+mock_data_blueprint = Blueprint('mock_Ddata', __name__)
+
+
+class MockDataApi(MethodView):
+    @requires_auth(allowed_basic_auth_types=('superadmin',))
+    def post(self):
+        post_data = request.get_json()
+
+        number_of_users = post_data.get('number_of_users')
+        number_of_transfers = post_data.get('number_of_transfers')
+        number_of_days = post_data.get('number_of_days')
+        amount_to_disburse = post_data.get('amount_to_disburse', 1000)
+
+        safety_check = post_data.get('safety_check')
+        total_users = User.query.execution_options(show_all=True).count()
+
+        if safety_check != total_users:
+            # Prevent a whole bunch of data being created unintentionally
+            response_object = {
+                'message': 'Failed Safety Check'
```

Comment:
IMO this error should be more descriptive. Maybe print what the safety check is, and possibly the correct `total_users` value

Reply:
hahaha I kinda left it ambiguous on purpose, to force the user to actually think carefully before firing it off. But on reflection it's probably fine to provide the info because it'll at least stop the api from being accidentally called. will amend